src/pages/iou/request/step/IOURequestStepDescription.tsx

```diff
@@ -89,15 +65,14 @@ function IOURequestStepDescription({
     reportActions,
     session,
     report,
-}) {
+}: IOURequestStepDescriptionProps) {
     const styles = useThemeStyles();
     const {translate} = useLocalize();
-    const inputRef = useRef(null);
-    const focusTimeoutRef = useRef(null);
+    const inputRef = useRef<AnimatedTextInputRef | null>(null);
+    const focusTimeoutRef = useRef<NodeJS.Timeout | null>(null);
     // In the split flow, when editing we use SPLIT_TRANSACTION_DRAFT to save draft value
     const isEditingSplitBill = iouType === CONST.IOU.TYPE.SPLIT && action === CONST.IOU.ACTION.EDIT;
-    const currentDescription =
-        isEditingSplitBill && !lodashIsEmpty(splitDraftTransaction) ? lodashGet(splitDraftTransaction, 'comment.comment', '') : lodashGet(transaction, 'comment.comment', '');
+    const currentDescription = isEditingSplitBill && !lodashIsEmpty(splitDraftTransaction) ? splitDraftTransaction?.comment.comment ?? '' : transaction?.comment.comment ?? '';
```

Comment:
```suggestion
    const currentDescription = isEditingSplitBill && !isEmptyObject(splitDraftTransaction) ? splitDraftTransaction?.comment.comment ?? '' : transaction?.comment.comment ?? '';
```

Couldn't we use `isEmptyObject` instead of `lodash` here?

Reply:
I missed that one, +1